modules/juce_core/native/juce_win32_SystemStats.cpp

```diff
@@ -166,6 +168,30 @@ void CPUInformation::initialise() noexcept
     hasAVX512VL        = (info[1] & (1u << 31)) != 0;
     hasAVX512VBMI      = (info[2] & (1u <<  1)) != 0;
     hasAVX512VPOPCNTDQ = (info[2] & (1u << 14)) != 0;
+#else
+    hasMMX = FALSE;

```

Comment:
Not sure why you're sticking the Win32 `FALSE` here instead of the C++ `false`? Regardless, those values will always be `false`; ARM doesn't have SSE or AVX registers.

Reply:
Yeah it's a bit of a bolt-on.  I think there could be some effort put into avoiding the interleaved target conditionals.  I see some arguments for having a separate file per platform.

Reply:
Bolt-on to achieve... what? I'm not sure I understand here.